src/common.speech/TranscriptionServiceRecognizer.ts

```diff
@@ -142,9 +166,15 @@ export class TranscriptionServiceRecognizer extends ConversationServiceRecognize
     // Encapsulated for derived service recognizers that need to send additional JSON
     protected async sendTranscriptionStartJSON(connection: IConnection): Promise<void> {
         await this.sendSpeechContext(connection, true);
-        const info: ConversationInfo = this.privTranscriberRecognizer.getConversationInfo();
-        const payload: { [id: string]: any } = this.createSpeechEventPayload(info, "start");
-        await this.sendSpeechEvent(connection, payload);
+        if (this.privTranscriberRecognizer.IsMeetingRecognizer()) {
+            const info: MeetingInfo = this.privTranscriberRecognizer.getMeetingInfo();
```

Comment:
This seems like unnecessary duplication since MeetingInfo and ConversationInfo are structurally identical. It may be a bit of a refactor, so for now is there a way to alias MeetingInfo and rename ConversationProperties to something like GroupProperties (so that it can be used in both MeetingInfo and ConversationInfo)?